internal/admin/exports/form_test.go

```diff
@@ -0,0 +1,25 @@
+package exports
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestSplitRegions(t *testing.T) {
+	tests := []struct {
+		r string
+		e []string
+	}{
+		{"", []string{}},
+		{"  test  ", []string{"test"}},
+		{"test\n\rfoo", []string{"test", "foo"}},
```

Comment:
Can we add a test for _just_ `\n` separated?

Reply:
Done.